setup.json

```
@@ -14,10 +14,10 @@
     ],
     "extras_require": {
         "pre-commit": [
-            "pre-commit==1.17.0",
-            "yapf==0.28.0",
-            "prospector==1.1.7",
-      	    "pylint==2.3.1"
+		"pre-commit==1.17.0",
+		"yapf==0.29.0",
```

Newest version is 0.30, but not sure if this fits here?
```suggestion
		"yapf==0.30.0",
```